geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/SimpleFeatureTypes.scala

```diff
@@ -35,6 +38,10 @@ import scala.util.parsing.combinator.JavaTokenParsers
 
 object SimpleFeatureTypes {
 
+  type FR = FeatureReader[SimpleFeatureType, SimpleFeature]
+  type DFR = DelegateFeatureReader[SimpleFeatureType, SimpleFeature]
+  type DFI = DelegateFeatureIterator[SimpleFeature]
```

Comment:
hmm, this doesn't seem like a great place for these... maybe move to a package object somewhere?